server/controllers/payroll/employees/index.js

```diff
@@ -387,3 +393,65 @@ function search(req, res, next) {
   .catch(next)
   .done();
 }
+
+/**
+ * @method find
+ *
+ * @description
+ * This function scans the employee table in the database to find all values
+ * matching parameters provided in the options parameter.
+ *
+ * @param {Object} options - a JSON of query parameters
+ * @returns {Promise} - the result of the promise query on the database.
+ */
+function find(options) {
+  // ensure epected options are parsed appropriately as binary
+  db.convert(options, ['grade_id', 'creditor_uuid', 'patient_uuid']);
+
+  const filters = new FilterParser(options, { tableAlias : 'e' });
```

Comment:
For best practice, set [autoParseStatements](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/server/lib/filter.js#L56) to `false`.  This option was added later to FilterParser (https://github.com/IMA-WorldHealth/bhima-2.X/commit/be9ebdcd68c22e0df178837d13baefd400b163d4) to ensure that the library didn't do anything the programmer had not explicitly requested.  It also helps the code read better.